src/test_driver/tizen/assets.gni

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("${build_root}/config/compiler/compiler.gni")
+import("${chip_root}/src/ble/ble.gni")
+
+# Declare arguments used in the build_example.py script (but not declared by
+# any of the imported files above), so we can reuse them for generaing default
+# values for the tests assets.
+declare_args() {
+  chip_enable_wifi = true
```

Comment:
chip_enable_wifi is declared in src/platform/device.gni

In any case, this is hackish: build_example.py should control what arguments make sense. This tries to work around build_examples being lazy in its logic. Lets not do that: add args that make sense and build_examples.py has to sort it out.

Reply:
I've removed the assets file.